include/mvd/mvd_generic.hpp

```diff
@@ -57,25 +56,31 @@ inline MVDFile* open_mvd(const std::string & filename) {
 /// performed, but no guarantees about file correctness are made.
 ///
 /// \param filename the path of the file to open
+/// \param extra_arg extra argument to open tsv file or sonata parameter
 /// \return a shared pointer to a mvd::File object
 ///
 inline std::shared_ptr<File> open(const std::string& filename,
-                                  const std::string& population="default") {
+                                  const std::string& extra_arg="default") {
```

Comment:
I wasn't sure about this.. The thing is that `population` is an optional parameter and if we overload `open` function with `inline std::shared_ptr<File> open(const std::string& mvd3filename, const std::string& tsvfilename, const std::string& population="default")` then I guess that there would be clash if we called `open("circuit.mvd3", "mecombo.tsv")`.
I didn't know that there might be a case that `population` could be used with `mvd3` files though, since there is no other use of `population` apart from the `sonata` file.
In this case should we avoid the optional argument or is there a better solution?

Reply:
What about having the TSV access (which should be optional anyway) explicitly done via
```
auto file = MVD::open("filename");
file->readTSVInfo("tsv_filename");
```
?

I think it's hard to be implicit here and not lead to surprises

Reply:
I was about to suggest the same. `openTSV` should be explicit. We could potentially append it to mvd3 or sonata

Reply:
Okay, this makes more sense. I will take care of it :+1: